src/main/groovy/com/ullink/Msbuild.groovy

```diff
@@ -44,6 +44,8 @@ class Msbuild extends ConventionTask {
             project.file(project.name + ".csproj").exists() ? project.name + ".csproj" : null
         }
         conventionMapping.map "projectName", { project.name }
+        
+        outputs.files { projects.collect { _, project -> project.dotnetArtifacts.files }.flatten() }
```

Comment:
I purposedly removed the outputs from this task as they aren't correct enough to be useful, did you test this ?